package.json

```diff
@@ -38,7 +38,6 @@
     "test:karma": "cross-env NODE_ENV=test TZ=UTC karma start test/karma.conf.js",
     "test:karma:parallel": "cross-env NODE_ENV=test TZ=UTC PARALLEL=true karma start test/karma.conf.js",
     "test:unit": "cross-env NODE_ENV=test TZ=UTC mocha -n expose_gc",
-    "test:charts:unit": "cross-env NODE_ENV=test TZ=UTC mocha -n expose_gc --config packages/x-charts/.mocharc.js",
```

Comment:
Tbh I would still rather have a separate test. I don't think it brings us any advantage having the packages all tested by the same command, it just makes things slower.

Reply:
That would help, but is not the only issue, there is a lot of setup and helpers that don't affect us, but still has to be loaded because we share the tests. 

The tests have nothing to do with each other, and should probably be ran completely separate as well.

I would argue we should have specific tests and configurations for each library, with the only the strictly necessary setup being shared 😅